File: py/IEC61970/Base/Domain/ActivePowerChangeRate.py
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Domain.UnitMultiplier import UnitMultiplier
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Domain.UnitSymbol import UnitSymbol


class ActivePowerChangeRate:
    def __init__(self):
        self.multiplier = UnitMultiplier.none  # The unit multiplier for the active power change rate value
        self.value: float = 0.0  # The numerical value of the active power change rate

    unit = UnitSymbol.WPers  # Static constant unit symbol for active power change rate

File: java/IEC61970/Base/Domain/ActivePowerChangeRate.java
package IEC61970.Base.Domain;


/**
 * Rate of change of active power per time.
 * @created 30-Dec-2023 4:19:37 PM
 */
public class ActivePowerChangeRate {

	public UnitMultiplier multiplier;
	public static final UnitSymbol unit = WPers;
	public Float value;

	public ActivePowerChangeRate(){

	}
}//end ActivePowerChangeRate